tests/cross/src/main/scala/tests/pc/BasePcDefinitionSuite.scala

```diff
@@ -1,59 +1,20 @@
 package tests.pc
 
 import tests.BasePCSuite
-import scala.meta.internal.metals.CompilerOffsetParams
-import org.eclipse.lsp4j.TextEdit
-import org.eclipse.{lsp4j => l}
-import scala.meta.internal.jdk.CollectionConverters._
-import scala.meta.internal.mtags.MtagsEnrichments._
-import scala.meta.internal.metals.TextEdits
 
 abstract class BasePcDefinitionSuite extends BasePCSuite {
+  val test: (String, String) => (String, String) =
```

Comment:
Can we rename `test` to something more meaningful? Maybe `runCheck`?

Reply:
Changed.